packages/target-ethers-v4/src/codegen/types.ts

```diff
@@ -40,6 +40,8 @@ export function generateInputType(evmType: EvmType): string {
       return 'string'
     case 'tuple':
       return generateTupleType(evmType, generateInputType)
+    case 'unknown':
```

Comment:
I think that `unknown` type should simply translate to `any`. 

As input parameter I think unknown doesn't make sense at all. Even for output I think it's just better to return `any` and do not thrive for 100% type correctness... but better DX, it's edge case anyway...  WDYT: @quezak

Reply:
`unknown` would be more correct, but also would be annoying to use, let's go with `any` as the generated type